messages/no.json

```diff
@@ -44,7 +44,12 @@
       "bachelor": "Bachelor",
       "master": "Master"
     },
-    "test": "test"
+    "bonus": {
+      "location": "Lokasjon",
+      "bonuses": "Bonuser",
```

Comment:
Lets name it "tidligere bonuser" of "bonus" in norwegian. I think it sounds a bit strange with bonuser

Reply:
@anemne  Then maybe similarly "previous bonuses" for English?

Reply:
![Screenshot 2024-12-18 at 09 16 58](https://github.com/user-attachments/assets/7099282e-066f-49d8-a4f1-e34d43b8f58b)
Looks 
![Screenshot 2024-12-18 at 09 17 03](https://github.com/user-attachments/assets/9bde7e65-05a0-42c2-9d36-deceac0b8477)
better 👌

Reply:
Agreed 👌